.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "search.exclude": {
```

Comment:
I think there are enough vs-code users in the NativeScript community that some sensible defaults for file exclusion are reasonable to include in the repo too. What do you think?